probe_ble_specification.rst

```diff
@@ -754,18 +754,21 @@ Food Safe Status
 The food safe status is expressed in a packed 4-byte field, indicating the current
```

Comment:
This is no longer a 4-byte field (this applies to the node docs as well).

Reply:
Great catch, thanks!